src/Execution/lbsCriterionBase.py

```diff
@@ -46,34 +46,34 @@
 
 import bcolors
 
-from src.Model.lbsProcessor import Processor
+from src.Model.lbsRank import Rank
```

Comment:
It seems you are not up to date with current develop branch. This change is already there.

Reply:
We were working I am afraid at the same time on the same thing. I had to resolve the conflicts manually.